torchaudio/prototype/rnnt_decoder.py

```diff
@@ -0,0 +1,342 @@
+from typing import Callable, Dict, List, Optional, NamedTuple, Tuple
+
+import torch
+
+from .rnnt import RNNT
+
+
+__all__ = ["RNNTBeamSearch"]
+
+
+class Hypo(NamedTuple):
```

Comment:
Since this is a public API, could you add docstring?
It might require `:ivar` to document attributes.
cf. https://github.com/pytorch/audio/blob/05ae795ae51c5f72ef7747cd3c06fadccff97e5a/torchaudio/backend/common.py#L1

Reply:
nit: Also, I feel it's better to spell out `Hypothesis`, but it's subjective.